api/api_errors.go

```diff
@@ -74,39 +96,80 @@ func ErrorIsIn(err error, errorTypes []error) bool {
 // ErrOutOfGas signals that a call failed due to insufficient gas.
 type ErrOutOfGas struct{}
 
-func (ErrOutOfGas) Error() string { return "call ran out of gas" }
+func (ErrOutOfGas) Error() string { return outOfGasDefaultMsg }
 
 // ErrActorNotFound signals that the actor is not found.
 type ErrActorNotFound struct{}
 
-func (ErrActorNotFound) Error() string { return "actor not found" }
+func (ErrActorNotFound) Error() string { return actorNotFoundDefaultMsg }
 
 type errF3Disabled struct{}
 
-func (errF3Disabled) Error() string { return "f3 is disabled" }
+func (errF3Disabled) Error() string { return f3DisabledDefaultMsg }
 
 type errF3ParticipationTicketInvalid struct{}
 
-func (errF3ParticipationTicketInvalid) Error() string { return "ticket is not valid" }
+func (errF3ParticipationTicketInvalid) Error() string { return ticketInvalidDefaultMsg }
 
 type errF3ParticipationTicketExpired struct{}
 
-func (errF3ParticipationTicketExpired) Error() string { return "ticket has expired" }
+func (errF3ParticipationTicketExpired) Error() string { return ticketExpiredDefaultMsg }
 
 type errF3ParticipationIssuerMismatch struct{}
 
-func (errF3ParticipationIssuerMismatch) Error() string { return "issuer does not match current node" }
+func (errF3ParticipationIssuerMismatch) Error() string { return ticketIssuerMismatchDefaultMsg }
 
 type errF3ParticipationTooManyInstances struct{}
 
-func (errF3ParticipationTooManyInstances) Error() string { return "requested instance count too high" }
+func (errF3ParticipationTooManyInstances) Error() string { return tooManyInstancesDefaultMsg }
 
 type errF3ParticipationTicketStartBeforeExisting struct{}
 
 func (errF3ParticipationTicketStartBeforeExisting) Error() string {
-	return "ticket starts before existing lease"
+	return ticketStartBeforeExistingDefaultMsg
 }
 
 type errF3NotReady struct{}
 
-func (errF3NotReady) Error() string { return "f3 isn't yet ready to participate" }
+func (errF3NotReady) Error() string { return f3NotReadyDefaultMsg }
+
+type ErrExecutionRevertedWithData struct {
```

Comment:
Now the dust has settled, what do you think about this just being `ErrExecutionReverted`? We don't have one already and we can just set the expectation that you always instantiate these with `data`.

Reply:
Yeah, This make sense, `withData` was added because we explicitly were returning `ErrorData()`, but now that's not the case. 

I will rename it.